packages/angular/.storybook/main.ts

```diff
@@ -1,20 +1,9 @@
-import type { StorybookConfig } from '@storybook/angular';
+import { type StorybookConfig } from '@storybook/angular';
 
 const config: StorybookConfig = {
```

Comment:
The Angular components/wrappers are not being loaded in the storybook?

<img width="1969" alt="image" src="https://github.com/user-attachments/assets/eb30bb3a-fb82-4b0d-8b58-c644e4b5ee53" />

Reply:
Hmm, apparently this happens on `main` as well. It worked fine when i ran Angular storybook locally. I'll investigate.

Reply:
Fixed it! Apparently you to explicit define a `constructor` and call `super()` when you extend the `CePassthrough` directive.